exist-core/src/main/java/org/exist/xquery/pragmas/TimerPragma.java

```diff
@@ -30,8 +30,8 @@
 import org.exist.xquery.value.Sequence;
 
 public class TimerPragma extends Pragma {
-
-    public  final static QName TIMER_PRAGMA = new QName("timer", Namespaces.EXIST_NS, "exist");
+    public final static String LOCAL_NAME = "timer";
```

Comment:
See my comment above about why renaming this to LOCAL_NAME is problematic and needs to be reversed.

Reply:
not renamed but added